plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -2584,6 +2584,23 @@ public function get_database_schema() {
 		$operational_data_table_name = $this->get_operational_data_table_name();
 		$meta_table                  = $this->get_meta_table_name();
 
+		/**
```

Comment:
Would it be worth adding this filter in it's own function in a utility/helper class as we may find more uses for it as we work through some of the other upcoming data storage projects?